src/pages/api/multipleProducts.ts

```diff
@@ -20,6 +20,7 @@ import { NumberOfProductsAttribute } from './howManyProducts';
 
 export interface BaseMultipleProductAttribute {
     products: MultiProduct[];
+    endTimesList?: string[];
```

Comment:
Does this need to be optional? Just thinking if it isnt, we can provide an empty array and use the length of that to decide if we need to use it, and that way we **know** it will be on the attribute, so we dont need to do the check for the presence of it.

Reply:
have done that was not sure if we used elsewhere but have removed it